package.json

```diff
@@ -15,7 +15,8 @@
     "precommit": "npm run lint",
     "start": "NODE_ENV=development BABEL_ENV=development webpack-dev-server --config=webpack/webpack.dev.config.js --progress",
     "test": "jest --coverage --passWithNoTests",
-    "travis-deploy-once": "travis-deploy-once"
+    "travis-deploy-once": "travis-deploy-once",
+    "update-snapshot": "jest --updateSnapshot"
```

Comment:
Is this the fix for you travis breakage issues you mentioned?

Reply:
No. The Travis issue from before was in Registrar and was addressed by [this](https://github.com/edx/registrar/pull/232) PR.

I added this command to be able to update Jest snapshots locally. Jest isn't installed globally, so I wasn't able to run `jest --updateSnapshot` on its own without using a script. I think it's a useful command to have to update screenshots.